### DISCUSSION TITLE
Support for Onvista Bank possible

### DISCUSSION THREAD

**Discussion by 0a1b** [ghostfolio/ghostfolio#2732] [Ideas]
First of all, thank you for the great work!

I didn't see the Onvista Bank among the supported platforms, so I wonder whether it could be added.

Cheers,
keep it up

**Comment by dtslvr** [comment]
Thank you for the feedback, @0a1b.

I have just added _onvista bank_ to [ghostfol.io](https://ghostfol.io).
